examples/getstarted/package.json

```diff
@@ -13,7 +13,7 @@
     "strapi": "strapi"
   },
   "dependencies": {
-    "@strapi/icons": "2.0.0-rc.21",
+    "@strapi/icons": "2.0.0-rc.22",
```

Comment:
shouldn't it be .23 now?

Reply:
This one is for 22, I have another PR for 23 https://github.com/strapi/strapi/pull/23384